content/tutorial/02-advanced-svelte/04-actions/01-actions/app-a/src/lib/App.svelte

```diff
@@ -1,7 +1,7 @@
 <script>
 	import Canvas from './Canvas.svelte';
 
-	const colors = ["rouge", "orange", "jaune", "vert", "bleu", "indigo", "violet", "blanc", "noir"].;
+	const colors = ['red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'violet', 'white', 'black'];
```

Comment:
C'est volontaire ? C'est le point qui posait problème peut-être ?
```suggestion
	const colors = ["rouge", "orange", "jaune", "vert", "bleu", "indigo", "violet", "blanc", "noir"];
```

Reply:
Même réponse que plus haut